### DISCUSSION TITLE
V2版本自定义快捷键

### DISCUSSION THREAD

**Discussion by luchenhao** [the1812/Bilibili-Evolved#2474] [功能建议 / Ideas]
官方的一些快捷键和我的插件有些冲突，比如D建，V1版本我通过控制台把冲突的按键设置空，可以取消官方的快捷键，V2设置了没有效果。
> bilibiliEvolved.settings.customKeyBindings = {
> one: 'd',
> two: 'r',
> fullscreen: 'f',
> webFullscreen: 'w',
> }

**Comment by luchenhao** [comment]
设置里留空，按键还是有效果。
![Snipaste](https://user-images.githubusercontent.com/16728827/140101651-f6fcbdb3-b943-4d47-b15f-eac414701ece.jpg)

**Reply by the1812** [reply]
似乎还不支持删除 b 站的快捷键, 需要有一个"空动作", 向它设定希望取消的快捷键应该就可以阻止原来的
